doc/Configuration.html

```diff
@@ -5863,7 +5863,7 @@
 >DefaultAddress</B
 >  [  <CODE
 CLASS="OPTION"
->dns-names|ip-addresses seperated with spaces</CODE
```

Comment:
This single `Configuration.html` is a legacy artifact, from a previous doc maintainer who used doc generation tools to generate the HTML, PDF, etc files.  It is no longer really maintained, and has become stale.  I've been contemplating removing it altogether, and instead wanting to move most (if not all) of the docs to Markdown.

Reply:
That makes sense to me.  As you can tell, I didn't even realize that there was multiple sections for each directive at the beginning.
It is why I later suggested moving directives to include files to ensure consistent messaging.
I was thinking it was there as a single downloadable help file, but then using include files would probably require some tool to combine it as a single file if you wanted to keep that format.
I doubt most people would be upset if you deprecated some of those single file help files.